usage/test-framework/etc/exampl-catalog.bom

```diff
@@ -0,0 +1,15 @@
+brooklyn.catalog:
```

Comment:
the files in `etc` at present don't look like they are used for anything.

maybe move them to `src/test/resources/test-framework-examples` with a `README.md` in that dir explaining what they are used for.  and maybe in the `qa` project we could run them all as integration tests?